packages/ckit/src/__tests__/secp256k1-sudt.spec.ts

```diff
@@ -819,6 +822,136 @@ test('deposit sudt cheque and claim it', async () => {
   eqAmount(await provider.getUdtBalance(receiver.getAddress(), sudt), 500);
 });
 
+test('exchange sudt for ckb', async () => {
+  const provider = new TestProvider();
+  await provider.init();
+
+  const { debug } = provider;
+  debug('exchange sudt for ckb');
+
+  const { SECP256K1_BLAKE160, SUDT } = provider.config.SCRIPTS;
+
+  const exchangeProviderPrivKey = randomHexString(64);
+  const exchangeProviderAddr = provider.parseToAddress({
+    hash_type: SECP256K1_BLAKE160.HASH_TYPE,
+    code_hash: SECP256K1_BLAKE160.CODE_HASH,
+    args: Secp256k1Signer.privateKeyToBlake160(exchangeProviderPrivKey),
+  });
```

Comment:
```suggestion
  const exchangeProviderSigner = provider.generateAcpSigner('SECP256K1_BLAKE160');
  const exchangeProviderAddr = exchangeProviderSigner.getAddress();
```

maybe it is easier to understand

Reply:
got it